gapis/api/transform/transformer.go

```diff
@@ -55,7 +55,8 @@ type Writer interface {
 	// State returns the state object associated with this writer.
 	State() *api.GlobalState
 	// MutateAndWrite mutates the state object associated with this writer,
-	// and it passes the command to further consumers.
+	// and it passes the command to further consumers. It must not survive
```

Comment:
"must not survive" is strange english. Maybe something like "Will panic if mutate fails with an error".

Reply:
Actually are we sure log.F does panic?

Reply:
Thanks for the English wording suggestion, applied.

log.F(ctx, **true**) will panic here: https://github.com/google/gapid/blob/ca1cb2f8866c17ef258050ad34332748cab0ebb2/core/app/log.go#L46
Because gapis is started as an app, here:
https://github.com/google/gapid/blob/ca1cb2f8866c17ef258050ad34332748cab0ebb2/cmd/gapis/main.go#L71